src/ubuntu/18.04/cross/arm64/hooks/pre-build

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$(dirname ${BASH_SOURCE[0]})/../../../../build-scripts/build-rootfs.sh ubuntu-16.04 xenial arm64 lldb3.9
```

Comment:
```suggestion
$(dirname ${BASH_SOURCE[0]})/../../../../build-scripts/build-rootfs.sh ubuntu-18.04 xenial arm64 lldb3.9
```

Reply:
Should `xenial` be changed to `bionic` as well?